modules/request-quota-increase/README.md

```diff
@@ -114,3 +99,18 @@ new_quotas = {
 When you run `terraform destroy` on this module, it does not affect your current quotas or your
 existing quota requests. In other words, you don't have to worry about quotas being reset to old
 values; once they have been increased, they stay that way!
+
+## Contributing
+
+When you want to add a new resource, you can check the available services with
```

Comment:
This sections needs a bit of context first: i.e., that AWS requires you to pass in special codes to request quotas, that there is no central place where these are all defined, that there are many of these, we've included the values for the IAM and VPC services in the `request-quota-increase` module already, but to add other services, we've created a script...